CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ Possible sections are:
 - fix link to CHANGELOG in CONTRIBUTING
 - fix spelling in bbox
 - escape backslash in README
+- fix typos
```

Comment:
I think we can rename the entry `fix spelling in bbox` to `fix various typos` to avoid too many similar changelog entries.

Reply:
solved!